web/src/beta/components/SettingsButtons/index.tsx

```diff
@@ -0,0 +1,50 @@
+import Icon from "@reearth/beta/components/Icon";
+import Text from "@reearth/beta/components/Text";
+import { styled, useTheme } from "@reearth/services/theme";
+
+export interface Props {
+  title: string;
+  icon: string;
+  onBlock?: () => void;
+  onEdit?: () => void;
+  onSetting?: () => void;
+}
+
+const SettingsButtons: React.FC<Props> = ({ title, icon, onBlock, onEdit, onSetting }) => {
+  const theme = useTheme();
+
+  return (
+    <Wrapper>
+      <StyledMainIcon size={16} onClick={onBlock} icon={icon} />
+      <Text
+        size={"xFootnote"}
+        color={theme.general.content.strong}
+        otherProperties={{ padding: "0px 4px" }}
+        onClick={onBlock}>
+        {title}
+      </Text>
+      <StyledSubIcon size={12} icon={"editIcon"} onClick={onEdit} />
+      <StyledSubIcon size={12} icon={"settings"} onClick={onSetting} />
```

Comment:
Could you add `cursor: pointer` or use `button` ?
I think the clickable areas are these 3
- StyleMenuIcon and Text
- StyledSubIcon
- StyledSubIcon